StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -296,6 +310,15 @@ extension NativeFlowController {
                         if !session.accounts.data.isEmpty || session.paymentAccount != nil
                             || session.bankAccountToken != nil
                         {
+                            self.delegate?.nativeFlowController(
```

Comment:
is it expected these only two of these if/else blocks need a `self.delegate?.nativeFlowController` call?

Reply:
yes, thank you for checking

note that the `self.delegate?.nativeFlowController` can be thought of just calling a function - just through a special "communication pattern" (delegates)

and in this case, it just happens that in only some of the cases we want to call a specific function (the explanation is a bit deep into implementation details)